pkgs/applications/misc/qownnotes/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchgit, qt5 }:
+
+let
+  version = "17.02.06";
+in
+stdenv.mkDerivation {
+  name = "qownnotes-${version}";
+
+  # needs git submodules, so can't use fetchFromGitHub
+  src = fetchgit {
+    rev = "refs/tags/linux-b2804-185423";
+    url = "https://github.com/pbek/QOwnNotes";
+    sha256 = "0fknc6ajnzcdsmxd9q15i23v2lbpcp3scgq51izclkg8yd9i9lys";
+  };
+
+  buildInputs = [ qt5.qtbase qt5.qtmultimedia qt5.qtscript ];
+
+  configurePhase = ''
```

Comment:
Please add `qmakeHook` to your `buildInputs` and remove the custom configure phase.